techeria_controller/templates/index.html

```diff
@@ -100,12 +102,14 @@
                                     {% if user.is_authenticated and not user.is_staff %}

```

Comment:
This change should fix the interface error, and an error on the cart page. If it doesn't we should take another look